backend/src/main/kotlin/app/ehrenamtskarte/backend/matomo/Matomo.kt

```diff
@@ -0,0 +1,165 @@
+package app.ehrenamtskarte.backend.matomo
+
+import app.ehrenamtskarte.backend.config.MatomoConfig
+import app.ehrenamtskarte.backend.config.ProjectConfig
+import app.ehrenamtskarte.backend.stores.webservice.schema.SearchParams
+import app.ehrenamtskarte.backend.verification.database.CodeType
+import app.ehrenamtskarte.backend.verification.database.repos.CardRepository
+import app.ehrenamtskarte.backend.verification.webservice.schema.types.CardGenerationModel
+import jakarta.servlet.http.HttpServletRequest
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.matomo.java.tracking.MatomoRequest
+import org.matomo.java.tracking.MatomoRequestBuilder
+import org.matomo.java.tracking.MatomoTracker
+import org.slf4j.LoggerFactory
+import java.io.IOException
+import java.util.concurrent.ExecutionException
+
+object Matomo {
```

Comment:
Optional: I think in this file there is some duplicated stuff which might be reduced.